src/component/cyfs-base/src/codec/raw/raw_types.rs

```diff
@@ -740,7 +740,7 @@ impl<'de, T: RawEncode + RawDecode<'de>> RawDecode<'de> for Vec<T> {
 
 // HashSet<T>
 
-impl<T: RawEncode> RawEncode for HashSet<T> {
+impl<T: RawEncode + std::cmp::Ord> RawEncode for HashSet<T> {
```

Comment:
After adding this `Ord` trait qualification, Values that do not support `Ord` trait cannot use this default codec anymore, But the impact is not too big